pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -54,20 +56,25 @@ in rustPlatform.buildRustPackage {
   pname = "tree-sitter";
   inherit src version cargoSha256;
 
-  buildInputs = lib.optionals stdenv.isDarwin [ Security ];
-
-  nativeBuildInputs = [ emscripten which ];
-
-  postPatch = ''
-    # needed for the tests
-    rm -rf test/fixtures/grammars
-    ln -s ${grammars} test/fixtures/grammars
+  buildInputs =
+    lib.optionals stdenv.isDarwin [ Security ];
+  nativeBuildInputs =
+    [ which ]
+    ++ lib.optionals webUISupport [ emscripten ];
```

Comment:
```suggestion
  buildInputs = lib.optionals stdenv.isDarwin [ Security ];

  nativeBuildInputs = [ which ]
    ++ lib.optionals webUISupport [ emscripten ];
```

Reply:
The newlines were pretty much intentional. :)